src/main/webapp/app/exercises/programming/manage/update/update-components/grading/programming-exercise-grading.component.html

```diff
@@ -0,0 +1,139 @@
+<section class="mb-5">
+    <h3 jhiTranslate="artemisApp.programmingExercise.wizardMode.detailedSteps.gradingStepTitle" id="artemisApp.programmingExercise.wizardMode.detailedSteps.gradingStepTitle"></h3>
+    <p jhiTranslate="artemisApp.programmingExercise.wizardMode.detailedSteps.gradingStepMessage"></p>
+    @if (isEditFieldDisplayedRecord().includeExerciseInCourseScoreCalculation) {
```

Comment:
Only added the `@if (isEditFieldDisplayedRecord()...` checks, displays more in the diff because the component was moved to a folder